Microsoft.DataTransfer.FunctionalTests/JsonFileFunctionalTests.cs

```diff
@@ -44,6 +44,12 @@ public async Task TransferFromJsonToJson_AllDataTransferred()
             var resultFile = new FileInfo(@"OutputData\Test1.json");
             Assert.IsTrue(resultFile.Exists, TestResources.OutputFileMissing);
             Assert.IsTrue(resultFile.Length > 0, TestResources.OutputFileEmpty);
+            using (var streamReader = new StreamReader(resultFile.FullName, System.Text.Encoding.UTF8))
+            {
+                string text = streamReader.ReadToEnd();
+                Assert.IsTrue(text.Contains("2017-12-22T09:00:00Z"), "Date processing discrepancy");
```

Comment:
How big is the file? You can probably add the whole thing and compare, to make sure we don't hit any other issues when somebody updates the JSON.NET next time.